app/views/school/home/show.html.erb

```diff
@@ -6,6 +6,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if FeatureFlag.active?(:christmas_banner) || FeatureFlag.active?(:increased_allocations_banner) %>
+      <%= render NotificationBanner.new(title: 'Important') do |banner| %>
+        <% if FeatureFlag.active?(:increased_allocations_banner) && @school.increased_allocations_feature_flag? %>
+          <p class="govuk-notification-banner__heading">
+            <%= t('banners.increased_allocations.school.heading', count: @school&.std_device_allocation&.allocation) %>
+          </p>
+          <p class="govuk-body">
+            <%= t('banners.increased_allocations.school.content') %>
+          </p>
+          <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```

Comment:
non-blocking remove this then afterwards put
```
        <% if FeatureFlag.active?(:christmas_banner) && FeatureFlag.active?(:increased_allocations_banner) %>
          <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
        <% end %>
```
otherwise there will be a small bug when Christmas flag comes off